src/js/degree-search-typeahead.js

```diff
@@ -0,0 +1,106 @@
+const degreeSearchInit = ($) => {
+  let degree;
+
+  if ($('.degree-search-typeahead')) {
```

Comment:
This if-statement will always return true (jquery will return an empty jquery object when the selection isn't found, which isn't false-y).

You can check the object's length instead, e.g. `if ($('.degree-search-typeahead').length) { ... }` to return false when the selection isn't found/returns an empty result.